### PR TITLE
[CI][Docker] Add aarch64 for ubuntu 20.04

### PR DESCRIPTION
**Pull request by 0yi0**

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review agent](https://github.com/flows-network/github-pr-review/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

**Comment by 0yi0** [comment]
Note: This check is not yet shown on the list https://github.com/WasmEdge/WasmEdge/actions/runs/10611454264

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -138,12 +138,13 @@ jobs:
...
-                {'name':'linux-static','compiler':'clang++','build_type':'Release','docker_tag':'ubuntu-build-clang','options':'-DWASMEDGE_BUILD_SHARED_LIB=Off -DWASMEDGE_BUILD_STATIC_LIB=On -DWASMEDGE_LINK_TOOLS_STATIC=On -DWASMEDGE_BUILD_PLUGINS=Off'},
-                {'name':'ubuntu-22.04-coverage','compiler':'g++','build_type':'Debug','docker_tag':'ubuntu-build-gcc','coverage':true,'tests':true}]"
+      matrix: "[{'name':'ubuntu-22.04','runner':'ubuntu-latest','compiler':'g++','build_type':'Debug','docker_tag':'ubuntu-build-gcc','tests':true},
+                {'name':'ubuntu-22.04','runner':'ubuntu-latest','compiler':'g++','build_type':'Release','docker_tag':'ubuntu-build-gcc','tests':true},
+                {'name':'ubuntu-22.04','runner':'ubuntu-latest','compiler':'clang++','build_type':'Debug','docker_tag':'ubuntu-build-clang','tests':true},
+                {'name':'ubuntu-22.04','runner':'ubuntu-latest','compiler':'clang++','build_type':'Release','docker_tag':'ubuntu-build-clang','tests':true},
+                {'name':'ubuntu-20.04','runner':'linux-arm-v2','compiler':'clang++','build_type':'Release','docker_tag':'ubuntu-20.04-build-clang-aarch64','tests':true},
```
Comment (hydai) [inline .github/workflows/build.yml:145]: It should be `linux-arm64-v2`. OAO

**Review by hydai — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (0yi0) [reply .github/workflows/build.yml]: QAQ

**Review by hydai — APPROVED** [review]

**Pull request merged**